apps/extension/src/hooks/numeraires-query.ts

```diff
@@ -3,26 +3,25 @@ import { ChainRegistryClient } from '@penumbra-labs/registry';
 import { useMemo } from 'react';
 
 export const useNumeraires = (chainId?: string) => {
-  const { data, isLoading, error, isError } = useQuery({
+  const { data, isLoading, isError } = useQuery({
     queryKey: ['registry', chainId],
     queryFn: async () => {
       const registryClient = new ChainRegistryClient();
       return registryClient.remote.get(chainId!);
     },
+    retry: 1,
+    retryDelay: 0,
```

Comment:
A single retry feels ok, without specifying these, react-query tends to retry indefinitely.

Reply:
Hmm. Maybe it's the delay then that causes such long retry times. Alas, for our case, we want to fail fast given this is not a critical setting to select.